force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -290,8 +316,10 @@ public with sharing class VariablesDataTypesOperators {
      * @param numberOfHours The number of hours worked in a week.
      * @return A Decimal representing the monthly paycheck.
      */
-    public static Decimal calculateMonthlyPaycheck(Decimal hourlyRate, Double numberOfHours) {
-        return null; // Replace null with the variable you used to store the result
+    public static Decimal calculateMonthlyPaycheck(Decimal hourlyRate, Decimal numberOfHours) {
+        Decimal weeklyPay = hourlyRate * numberOfHours;
+        Decimal monthlyPay = weeklyPay * 4;
+        return monthlyPay;
```

Comment:
Well done,
For Practice, try using a variable for the 4 and instead of creating a new weeklyPay, use the one from question 18